CHANGELOG.rst

```diff
@@ -1,5 +1,42 @@
 Changelog
 =========
+1.19.1.dev0 (2021-08-25)
+------------------------
+MLflow 1.19.1.dev0 includes several major features and improvements:
+
+Features:
```

Comment:
@harupy @WeichenXu123 Can we re-order the major features so that post-training metrics and pip dependency inference appear at the top?